App/HostedReactor.fs

```diff
@@ -0,0 +1,67 @@
+module HostedReactor
+
+open System
+open Microsoft.Extensions.Hosting
+open Propulsion.Internal
+open Propulsion.MessageDb
+
+type HostedReactor(connectionString, log, service: InvoiceNumberingReactor.Service, checkpointService: CheckpointStore.CheckpointService) =
+  inherit BackgroundService()
+
+  let stats =
+    { new Propulsion.Streams.Stats<_>(log, TimeSpan.FromMinutes 1, TimeSpan.FromMinutes 1) with
+        member _.HandleOk x = ()
+        member _.HandleExn(log, x) = () }
+
+  [<Literal>]
+  let groupName = "InvoiceNumberingReactor"
+
+  let categories = [| Invoice.Category |]
+
+  let checkpoints = CheckpointStore.CheckpointStore(checkpointService, groupName, TimeSpan.FromSeconds 10)
+  let maxReadAhead = 100
+  let maxConcurrentStreams = 2
+
+  let handle struct(stream, events) =
```

Comment:
-rc provides a task/func layer (without `struct`)

Reply:
Makes sense, I can update once https://github.com/jet/propulsion/pull/198 is merged

Reply:
Thanks/sorry. Will ping over there when its releasing

Reply:
No need to be sorry. My test didn't cover it, so it's entirely on me 🙈